openstack/nova/alerts/openstack/vmware-rebalancer.alerts

```diff
@@ -0,0 +1,17 @@
+groups:
+  - name: vmware-rebalancer.alerts
+    rules:
+      - alert: VcenterLowNumberOfPlaceableVms
+        expr: wm_vmware_rebalancer_placeable_vms_per_vcenter{has_bbs_with_custom_hana_exclusive_host="False",has_bbs_without_custom_hana_exclusive_host="True",flavor="g_c128_m512"} <= 3
+        for: 30m
+        labels:
+          severity: info
+          service: compute
+          support_group: compute
+          tier: vmware
+          context: "vCenter Capacity {{ $labels.vcenter }}"
+          meta: "The vCenter `{{ $labels.vcenter }}` is (almost) full and only allows scheduling less than 4 VMs."
+          playbook: docs/support/playbook/nova/alerts/alert-vcenter-full.md
```

Comment:
if the playbook doesn't exist, imho don't add one. compute colleagues might want to have it in their namespace in the docs..